package.json

```diff
@@ -25,8 +25,10 @@
     "file-loader": "0.8.5",
     "gulp": "3.9.1",
     "gulp-cache": "0.4.4",
+    "gulp-cached": "^1.1.0",
```

Comment:
We do not commit dependancies with ^ because we want to solidly nail down versions. Instead use the `general-update` script in frontend/bin (I have it on my $PATH)